common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4196,6 +4199,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     LLAP_LRFU_BP_WRAPPER_SIZE("hive.llap.io.lrfu.bp.wrapper.size", 64, "thread local queue "
         + "used to amortize the lock contention, the idea hear is to try locking as soon we reach max size / 2 "
         + "and block when max queue size reached"),
+    LLAP_IO_MAX_CLOCK_ROTATION("hive.llap.io.clock.max.rotation",
```

Comment:
Agree with you this is not for customer at all, it is more for dev nob in case of issue we can push the nob on the customer side without deploying code. Will add more comments about it. this makes me thing that we need to add some annotation about what configs are suppose to be dev configs and the one that can be used by customer. Any other suggestion will be welcomed.